extensions/api/v1alpha1/sandboxclaim_types.go

```diff
@@ -37,6 +37,33 @@ import (
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 // Important: Run "make" to regenerate code after modifying this file
 
+// DeletionPolicy describes the policy for deleting the SandboxClaim when it expires.
+// +kubebuilder:validation:Enum=Delete;Retain
+type DeletionPolicy string
+
+const (
+	// DeletionPolicyDelete deletes the SandboxClaim (and cascadingly the Sandbox) when expired.
+	DeletionPolicyDelete DeletionPolicy = "Delete"
+
+	// DeletionPolicyRetain keeps the SandboxClaim when expired (Status will show Expired).
+	DeletionPolicyRetain DeletionPolicy = "Retain"
+)
+
+// Lifecycle defines the lifecycle management for the Sandbox.
+type Lifecycle struct {
+	// ShutdownTime is the absolute time when the sandbox expires.
+	// +kubebuilder:validation:Format="date-time"
+	// +optional
+	ShutdownTime *metav1.Time `json:"shutdownTime,omitempty"`
+
+	// DeletionPolicy determines if the SandboxClaim (resource and underlying resources
+	// such as Sandbox, Pod, Service) itself should be
+	// deleted when the Sandbox expires.
```

Comment:
Current implementation lets the SandboxClaim shutdown time overwrites the Sandbox's, i.e. the Sandbox might be deleted before its expiration time. Might be good to mention is here as well. 

And this needs to be fixed:
```suggestion
	// deleted when the SandboxClaim expires.
```